scripts/dts/extract/globals.py

```diff
@@ -350,6 +398,15 @@ def extract_controller(node_path, prop, prop_values, index,
             continue
 
         cell_parent = phandles[elem[0]]
+
+        if 'gpio-map' in reduced[cell_parent]['props']:
+            # Parent is a gpio 'nexus node' (ie has gpio-map).
+            # Controller should be found in the map, using elem[1] as index.
+            # Pin attribues (number, flag) will not be used in this function
+            cell_parent, unused = child_to_parent_unmap(cell_parent, elem[1])
```

Comment:
Call it `_` instead of `unused`. That's common in Python, and `pylint` knows not to flag it.

Reply:
ok, thanks for the hint.